library/ssl_tls13_server.c

```diff
@@ -564,6 +564,9 @@ static int ssl_tls13_parse_pre_shared_key_ext(
 #if defined(MBEDTLS_SSL_SESSION_TICKETS)
         mbedtls_ssl_session session;
         mbedtls_ssl_session_init(&session);
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+        session.max_early_data_size = ssl->conf->max_early_data_size;
```

Comment:
I think this value would be overwriten after `ticket_parse`, and later would be used to overwrite `ssl->session_negotiate->max_early_data_size` by `ssl_tls13_session_copy_ticket` at L657. 

Pls check if this is expected.

Reply:
Agree when receiving a ticket we should use the `max_early_data_size` of that ticket. Thus I do not thing we need this assignment.